Update ansible_tips_tricks.rst

theputta opened the pull request:
Include information about using Ansible Lint in staging environment

Commit:
Update ansible_tips_tricks.rst

Include information about using Ansible Lint in staging environment

oraNod commented on docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst:
```diff
@@ -128,7 +128,11 @@ Try it in staging first
 
 Testing changes in a staging environment before rolling them out in production is always a great idea.
-Your environments need not be the same size and you can use group variables to control the differences between those environments.
+Your environments need not be the same size and you can use group variables to control the differences between those environments. you can also check for any syntax errors in staging environment using Ansible lint 
```
```suggestion
Your environments need not be the same size and you can use group variables to control the differences between those environments.
You can also check playbooks for syntax errors in staging environments with Ansible Lint, as follows:
```

felixfontein replied:
There's no need to use ansible-lint for checking for syntax errors. You can simply use `ansible-playbook --syntax-check` (which is what ansible-lint uses under the hood).

I would rather reference that instead of taking out the big hammer (i.e. ansible-lint).

Commit:
Update ansible_tips_tricks.rst

Updated based on recommendations in 

https://github.com/ansible/ansible-documentation/pull/275

theputta commented:
Made the recommended changes as per @samccann review

samccann commented on docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst:
```diff
@@ -128,7 +128,11 @@ Try it in staging first
 
 Testing changes in a staging environment before rolling them out in production is always a great idea.
-Your environments need not be the same size and you can use group variables to control the differences between those environments.
+Your environments need not be the same size and you can use group variables to control the differences between those environments. you can also check for any syntax errors in staging environment using flag --syntax-check 
```
```suggestion
Your environments need not be the same size and you can use group variables to control the differences between those environments. you can also check for any syntax errors in staging environment using flag ``--syntax-check``
```

samccann committed:
Update docs/docsite/rst/tips_tricks/ansible_tips_tricks.rst

samccann commented:
Thanks @theputta for the Ansible docs fix!
